modules/openapi-generator/src/main/resources/Java/libraries/microprofile-kumuluzee/pojo.mustache

```diff
@@ -0,0 +1,129 @@
+import javax.xml.bind.annotation.XmlElement;
```

Comment:
Enclose all the imports of 'javax.xml.bind.annotation' with in withXml because we don't need these imports to be part of generated class with withXml=false
{{#withXml}}
import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;
import javax.xml.bind.annotation.XmlEnum;
import javax.xml.bind.annotation.XmlEnumValue;
 {{/withXml}}